src/io/flutter/view/InspectorPanel.java

```diff
@@ -518,9 +524,11 @@ public void showProperties(DiagnosticsNode diagnostic) {
           // be loaded.
           return;
         }
+
+        properties.sort(Comparator.comparing(DiagnosticsNode::getName));
```

Comment:
If we sort by name, we should still show high priority properties first over debug level properties.

Reply:
How common is the different levels of priorities? Are the priorities for a property always the same or can they change (like changing to an error)?

I like lexical sorting as it's always a consistent order, and predictable for the user (they'll know where a property is in the list w/o having to know its priority). I'm wondering if we should instead call out importance via color/icon/styles instead of order.